### ISSUE TITLE
Debug gone wrong

### ISSUE THREAD

**Issue by PKNessTorio** [burnedpopcorn/Pizza-Tower-1.1.0-Web-Port#39]
Some of the debug commands don't _work_. I tried going to the old pt tutorial or turning into Gustavo, but **it ain't workin'!** _Please_ fix this.

**Comment by burnedpopcorn** [comment]
Yeah, for some reason letters are forced to be Uppercase, so I had to change debug commands to accommodate thus weird limitation

But because of how the Commands with multiple arguments work, I won't be able to this fix to those, so this won't be fixed for a while

**Comment by burnedpopcorn** [comment]
So the commands with multiple arguments are the ones that don't work

Commands like testprank or panic still work fine

**Comment by PKNessTorio** [comment]
Can you just change it so it _can_ work?

**Comment by Sammyboi120310** [comment]
How would you type out panic in the final round?
